src/window_background/updateDeck.js

```diff
@@ -5,10 +5,10 @@ import forceDeckUpdate from "./forceDeckUpdate";
 import getOpponentDeck from "./getOpponentDeck";
 import { objectClone } from "../shared/util";
 
-var lastDeckUpdate = new Date();
+const lastDeckUpdate = new Date();
```

Comment:
`update_deck` *should* update this later on, it's just buggy.

Making it `const` makes the bug seem intentional.

```suggestion
let lastDeckUpdate = new Date();
```